languages/erlang/exercises/concept/basics/.docs/hints.md

```diff
@@ -0,0 +1,3 @@
+- You can use [arithmetic operators](http://erlang.org/doc/reference_manual/expressions.html#arithmetic-expressions) to implement the functions
```

Comment:
When possible, links are linked to individual tasks. If that doesn't make sense for a particular hint, they can be a general hint. The format in which the hints.md file should be written is slightly different, where there are headings added (see [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docshintsmd)). For an example of such a hints file, see [this file](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/floating-point-numbers/.docs/hints.md).